server.js

```diff
@@ -706,6 +710,16 @@ module.exports.initExpress = function() {
         require('./pages/studentAssessmentInstanceHomework/studentAssessmentInstanceHomework'),
         require('./pages/studentAssessmentInstanceExam/studentAssessmentInstanceExam'),
     ]);
+    app.use('/pl/course_instance/:course_instance_id/instructor/assessment/:assessment_id/assessment_question/:assessment_question_id/manual_grading', [
```

Comment:
Again pet peeve: this uses "manual_grading" in the same way as the manual_grading page above, which from the URL sounds like this will list all manual grading questions. Suggestion: "next_ungraded".

Reply:
Agreed. Done.